constph.py

```diff
@@ -55,9 +55,17 @@
 import simtk.unit as units
 import pymbar
 
+def strip_unit(quant):
+    """Strips the unit from a simtk.units.Quantity object and returns it's value in OpenMM consistent units.
+        Returns quant if object is not a quantity."""
+    if units.is_quantity(quant):
+        return quant.in_unit_system(units.md_unit_system)._value
```

Comment:
You can replace this with `value_in_unit(units.md_unit_system)`.  Probably better than using the `_value` attribute.  The only time I use `_value` directly is when I want to keep the number as a unit, but explicitly bypass dimensional analysis.